petals-esb-deployer-impl/src/test/java/org/ow2/petals/deployer/utils/ModelDeployerImplTest.java

```diff
@@ -47,4 +49,17 @@ public void deployModel() throws Exception {
 
         verify(runtimeModelDeployerMock).deployRuntimeModel(any(RuntimeModel.class));
     }
+
+    @Test
+    public void deployModelWithMavenUrl() throws Exception {
+        System.setProperty("java.protocol.handler.pkgs", "org.ops4j.pax.url");
+
+        try {
+            modelDeployer.deployModel(ModelUtils.generateTestModelMavenUrl());
```

Comment:
Why to not add `verify(runtimeModelDeployerMock).deployRuntimeModel(any(RuntimeModel.class));` as for other unit tests ?

Reply:
So if this test fails, we know it is caused by maven urls.